apps/sam-design-system-site/src/app/app.module.ts

```diff
@@ -26,14 +27,20 @@ export function hljsLanguages() {
 }
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent,SdsToastComponent],
```

Comment:
Is there any reason why we need to add a component in the app-level..?

Reply:
refuses to work any other way